format/view-spec.md

```diff
@@ -82,9 +98,13 @@ Each version in `versions` is a struct with the following fields:
 | _required_  | `representations`   | A list of [representations](#representations) for the view definition         |
 | _optional_  | `default-catalog`   | Catalog name to use when a reference in the SELECT does not contain a catalog |
 | _required_  | `default-namespace` | Namespace to use when a reference in the SELECT is a single identifier        |
+| _optional_  | `storage-table`     | A [partial identifier](#partial-identifier) of the storage table |
 
 When `default-catalog` is `null` or not set, the catalog in which the view is stored must be used as the default catalog.
 
+When 'storage-table' is `null` or not set, the entity is a common view, otherwise it is a materialized view. 
+The catalog of the storage table must be same as the one of the materialized view.
```

Comment:
I thought the consensus is either the catalog is the same as the one of the MV (implicit), or the catalog name can be specified explicitly too. 

portability across engines is assumed with consistent catalog naming across engines.

Reply:
I've updated the document accordingly.

Reply:
@stevenzwu @JanKaul @danielcweeks Does this overlap with https://github.com/apache/iceberg/pull/11365? Can we discuss both in a wholistic way?